NOTES.UNIX

```diff
@@ -8,35 +8,53 @@
  Shared libraries and installation in non-standard locations
  -----------------------------------------------------------
 
- Binaries on Unix variants expect to find shared libraries in standard
- locations, such as /usr/lib, /usr/local/lib and some other locations
- configured in the system (for example /etc/ld.so.conf on some systems).
- If the libraries are installed in non-standard locations, binaries
- will not find them and therefore fail to run unless they get a bit of
- help from a defined RPATH or RUNPATH.  This can be applied by adding
- the appropriate linker flags to the configuration command, such as
- this (/usr/local/ssl was the default location for OpenSSL installation
- in versions before 1.1.0):
+ Binaries on some Unix variants expect to find shared libraries in
+ standard locations, such as /usr/lib and possibly /usr/local/lib.  If
+ the libraries are installed in non-standard locations, binaries will not
+ find them and therefore fail to run unless they get a bit of help from
+ a defined runtime shared library search path.
+
+ It is possible to specify additonal directories for the runtime shared
+ library search path when linking a shared library or an application.
```

Comment:
I'd rather say that if OpenSSL shared libraries are installed in non-default locations, non-default from OS vendor viewpoint, then it would be more than appropriate to specify additional directories. What I'm saying is that it should sound rather like advice to specify additional directories, than a mention of mere possibility.

Above you mention /usr/lib, /usr/local/lib. I really don't feel that one should call them "standard". As example on Tru64 standard location is /shlib. Even on Linux primary location is actually /lib. I'd say that "conventional" probably better captures the nature of /usr/lib /usr/local/lib.

Reply:
*primary* doesn't mean *standard*...  On Linux, `/lib` and `/usr/lib` are both  standard as far as I know, just with different uses.  But hey, the `ld.so` manual talks about *default* directories, perhaps that's a better term?

Reply:
Re "possible", it suggests that for users who need to set rpath, it's possible to do with our configuration scripts.  I'll see if I can word it differently.